dotcom-rendering/src/web/components/App.tsx

```diff
@@ -1286,7 +1286,18 @@ export const App = ({ CAPI, NAV, ophanRecord }: Props) => {
 				</Lazy>
 			</Portal>
 			<Portal rootId="sign-in-gate">
-				<SignInGateSelector isSignedIn={isSignedIn} CAPI={CAPI} />
+				<SignInGateSelector
+					isSignedIn={isSignedIn}
+					format={format}
+					contentType={CAPI.contentType}
+					sectionName={CAPI.sectionName || ''}
```

Comment:
Elsewhere in this PR (for `host`) we've switched to specifying the default _inside_ the component, rather than at the call site. Do we want to do the same here for consistency?

Reply:
Great spot! Yes, I think it makes more sense to have all the defaults in one place